app/components/find/courses/contact_details_component/view.html.erb

```diff
@@ -3,11 +3,20 @@
 
   <div class="course-basicinfo">
     <dl class="app-description-list">
-      <% if course.provider.email.present? %>
-        <dt class="app-description-list__label">Email</dt>
+       <!-- Temporary stop gap for https://trello.com/c/vJRKI35V/614-stop-gap-ucl-is-moving-away-from-using-email. BIN it as soon as we have a better solution. -->
+      <% if course.provider.provider_code == "U80" %>
```

Comment:
@JR-G @MylesJarvis we probably want to card these to look at in the future. We have random provider codes in the codebase, due to having made "exceptions" for providers, like this one...